ocaml/xapi/xapi_ha_vm_failover.ml

```diff
@@ -555,33 +555,40 @@ let compute_anti_aff_evac_plan ~__context total_hosts hosts vms =
          anti-affinity evacuation plan is not needed." ;
       binpack_plan ~__context config vms
   | _ ->
-      let anti_aff_vms, non_anti_aff_vms = vms |> vms_partition ~__context in
-      let spread_evenly_plan_pool_state =
-        init_spread_evenly_plan_pool_state ~__context anti_aff_vms hosts
-      in
-      let anti_aff_vms_increasing =
-        anti_aff_vms |> List.sort (fun (_, a, _) (_, b, _) -> compare a b)
-      in
+      if Pool_features.is_enabled ~__context Features.VM_groups then
```

Comment:
Maybe match it together with total_hosts, like:
match (Pool_features.is_enabled ~__context Features.VM_groups, total_hosts) would make it more clear.